docs/UserGuide.md

```diff
@@ -300,6 +300,9 @@ You can use this command to add a new tutorial lesson for a specified tutorial g
 
 `addlsn grp/GROUP_NAME lsn/LESSON_NAME`
 
+> :memo: LESSON_NAME follows the format 'X-Y' where X is the week number, and Y is the lesson number. There can be 
+>more than 9 weeks (as in 10-1) but not more than 9 lessons in a week (1-10 would be invalid).
```

Comment:
There can be 10 lessons in a week (lesson 1-0 to 1-9)
So on a side note, should we allow users to create lesson 1-0?

Reply:
Good catch! I changed it to allow only lessons 1-1 to 1-9, so it's now 9 lessons.
Added test case for it too.